### DISCUSSION TITLE
I3 bindsym with rofi is giving the "Rofi doesn't know what to show" error.

### DISCUSSION THREAD

**Discussion by HydraJkm** [i3/i3#6290] [Q&A] [answered]
My command inside of the config is 'bindsym $mod+r exec --no-startup-id rofi -show drun' My locales should be fine. Rofi works fine when ran from terminal.

**Comment by okraits** [comment]
Your keybinding works fine here. Which versions of i3 and rofi are you using?

**Reply by HydraJkm** [reply]
Version 1.7.5+wayland3 of rofi. Version 4.23 of i3.

**Reply by HydraJkm** [reply]
Tried to uninstall the wayland version and install normal version, but now it doesn't give any error, and just doesn't do anything.

ETA: Still 1.7.5 rofi

**Comment by HydraJkm** [comment, marked as answer]
The answer to this problem seems to be my own foolishness. Having also using hyprland, I assumed just writing the file would also reload i3. In reality, it is required you press the specified bind (default being $mod+Shift+c).
